src/demo-app/datepicker/datepicker-demo.ts

```diff
@@ -98,3 +101,35 @@ export class CustomHeader<D> implements OnDestroy {
         this._dateAdapter.addCalendarYears(this._calendar.activeDate, 1);
   }
 }
+
+@Component({
+    moduleId: module.id,
+    selector: 'customer-header-ng-content',
+    template: `
+        <mat-calendar-header #header>
+            <button mat-button type="button" class="mat-calendar-today-button"
+                    (click)="todayClicked()" [attr.aria-label]="todayButtonLabel"
+                    cdkAriaLive="polite">
```

Comment:
The button shouldn't have `cdkAriaLive`.

Reply:
Should I remove `attr.aria-label` as well? To be honest I was just following suit based on the buttons in `mat-calendar-header`

(https://github.com/angular/material2/blob/master/src/lib/datepicker/calendar-header.html#L4-L5)

Reply:
I've gone ahead and removed it for brevity